weave/tests/test_op_decorator_behaviour.py

```diff
@@ -132,7 +132,7 @@ def test_sync_method_call(client, weave_obj, py_obj):
             entity="shawn",
             project="test-project",
             name="A",
-            digest="nCFXG0JUooNi5QBQGsOPNhmNVRr8ueUKvEUDQ2TeSoc",
+            digest="tGCIGNe9xznnkoJvn2i75TOocSfV7ui1vldSrIP3ZZo",
```

Comment:
this and other digest changes are due to the project id change from hexdigest to base64 - this does not indicate a change in the underlying digest logic.